src/components/MainContainer/MainContainer.js

```diff
@@ -1,11 +1,14 @@
-import React, { Component } from 'react'
-import MoviesList from './MoviesList'
-import MoviesData from '../data/moviesList'
-import Search from './Search'
-import SortSearch from './SortSearch'
-import SortByCategory from './SortByCategory'
+import React, { Component } from "react"
+import MoviesList from '../MoviesList/MoviesList'
+import MoviesData from '../../data/moviesList'
+import Search from '../Search/Search'
+import SortSearch from '../SortSearch/SortSearch'
+import SortByCategory from '../SortByCategory/SortByCategory'
+import Header from '../Header/Header'
```

Comment:
please add `index.js` file to have import like `import Header from '../Header'`;

in addition it will be great to support absolute imports

Reply:
Updated all the Component files as index.js now